opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -412,21 +412,45 @@ namespace Opm {
         return m_doubleGridProperties.hasKeyword( keyword );
     }
 
-    /*
-      1. The public methods getIntGridProperty & getDoubleGridProperty will
-         invoke and run the property post processor (if any is registered); the
-         post processor will only run one time.
-
-         It is important that post processor is not run prematurely, internal
-         functions in EclipseState should therefore ask for properties by
-         invoking the getKeyword() method of the m_intGridProperties /
-         m_doubleGridProperties() directly and not through these methods.
-
-      2. Observe that this will autocreate a property if it has not been
-         explicitly added.
-    */
+
     const GridProperty<int>& Eclipse3DProperties::getIntGridProperty( const std::string& keyword ) const {
-        return m_intGridProperties.getKeyword( keyword );
+        auto& gridProperty = const_cast< Eclipse3DProperties* >( this )->m_intGridProperties.getKeyword( keyword );
```

Comment:
Can you flip the order of testing ACTNUM?  I.e., start the function with

    if (keyword != "ACTNUM")
        return m_intGridProperties.getKeyword(keyword);

And if not, _then_ do `const_cast`?